SeaBattle/Sea.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeaBattle
+{
+    public class Sea
+    {
+        private List<Deck> Decks { get; set; }
+        private List<BaseShip> Ships { get; set; }
+        int SeaWidth;
+        int SeaLenght;
+
+
+
+        public void Create()
+        {
+            //will finish this IF parts tommorow. 
+            //if ( > 10 &&  > 10)
+            //{
+            //}
+
+            for (int i = 0; i < Ships.Count; i++)
+            {
+                for (int j = 0; j < Decks.Count; j++)
+                {
+
+                }
+
+            }
+
+            var items = new List<PointStruct>(4);
```

Comment:
You still using PointStruct, while you replaced it with native Point struct, it probably doesn't even work, since I don't see it in the branch.

Reply:
fixed.